spec/amp-versioning-policy.md

```diff
@@ -1,61 +1,69 @@
-- Keep all users of AMP on secure versions of the library.
+- Keep all users of AMP on secure and updated versions of the library.
+- Enable all AMP developers to keep their AMP pages up-to-date.
+
+## Versioning
+
+- The AMP JS library and its extensions are versioned. There is also an **[AMP release](https://github.com/ampproject/amphtml/releases/)** that corresponds to the precise code. Each major and minor version may correspond to many AMP releases.
+- AMP is **released** about once per week and each release has a unique version number. With each release, AMP pages are rendered with the latest code for the respective major/minor versions of the AMP JS library and extensions.
+
+### AMP JS library
+
+The AMP JS library is distributed via its major version only. The major version is expected to change rarely if ever (on the order of less than once per year).
+
+### AMP extensions
 
-## Non-breaking changes
+Each AMP extension is versioned independently. Publishers of AMP documents can choose the major and minor version of an extension; they cannot choose the patch version. 
 
-- If a change is backward compatible, no version should be changed.
-- This means that, for now, there is no known case under which the minor version of an extension would be changed.
+- Minor versions may be changed for breaking behavior changes *without* breaking API changes.
+- Major versions *must* be changed for any breaking API changes.
 
 ## Breaking changes
 
-Breaking changes are:
+Breaking changes must be implemented in a new version. Breaking changes are defined as:
 
-- Changes to the semantics of supported attributes and other exposed APIs.
+- Changes to exposed API semantics, e.g. supported attributes in an AMP extension.
 - Changes to CSS included in AMP that are not backward compatible.
 - Changes to the DOM structure (elements, attributes and attribute values) generated by AMP extensions that are not backward compatible.
 
 The following special cases are not considered breaking changes:
 
 - Changes to elements and their children including their CSS if the element name starts with `i-`.
-- Changes to attribute and class names starting with `-`. But this doesn't mean, CSS backward compatibility may be broken with this mechanism, where it would otherwise be considered a breaking change.
+- Changes to attribute and class names starting with `i-` (but this doesn't mean that CSS backward compatibility may be broken with this mechanism, which would be considered a breaking change).
 - Changes required to maintain the security of AMP pages.
-- Changes that break fewer than 0.1% of crawler accessible AMP pages.
+- Changes that break fewer than 0.1% of crawler-accessible AMP pages.
 
-For breaking changes the major version of an extension (or more rarely) the main AMP JS library should be increased.
+For breaking changes, the major version of an extension (or more rarely) the main AMP JS library should be increased.
 
-## Deprecation process
+## Deprecations
```

Comment:
How do deprecations that are breaking interact with the rules for breaking changes above?

If the deprecation process is followed to introduce a breaking change, are the same requirements still at play (security -or- < 0.1%), or does the deprecation process represent an alternative route / exception to making breaking changes?

Reply:
The latter. Made this clearer, thanks.